flash-gateware.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+title: flash-gateware
+published: true
+tags: hardware, numato-opsis
+---
+
+# flash-gateware
+
+Known Problems 
```

Comment:
See https://github.com/timvideos/HDMI2USB-mode-switch/issues/33 and https://github.com/timvideos/HDMI2USB-litex-firmware/blob/master/getting-started.md#common-errors for some more detailed instructions.

Reply:
those go here:  "problems and solutions"